### PR TITLE
Add initial commit for selenide AA

### PR DESCRIPTION
**Pull request by karanshah-browserstack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+To install the dependencies for iOS tests, run :
+
+```sh
+cd ios/
+mvn clean
+```
+
+## Getting Started
+
+Getting Started with Appium tests in TestNg on BrowserStack couldn't be easier!
```
Comment (sriteja777) [inline README.md:39]: Selenide?

File: android/src/test/java/com/browserstack/AppiumTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+public class AppiumTest {
+
+    public AndroidDriver driver;
+
+    @BeforeMethod(alwaysRun=true)
+    public void setUp() throws Exception {
+        MutableCapabilities capabilities = new UiAutomator2Options();
+        driver = new AndroidDriver(new URL("http://127.0.0.1:4723/wd/hub"),capabilities);
```
Comment (sriteja777) [inline android/src/test/java/com/browserstack/AppiumTest.java:20]: do we want to add source if missing?

File: .github/workflows/maven-workflow-run.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            repo: context.repo.repo,
+            name: process.env.job_name,
+            head_sha: process.env.commit_sha,
+            status: 'completed',
+            conclusion: process.env.conclusion
+            }).catch((err) => ({status: err.status, response: err.response}));
+            console.log(`The status-check response : ${result.status} Response : ${JSON.stringify(result.response)}`)
+            if (result.status !== 201) {
+              console.log('Failed to create check run')
+            }
```
Comment (khrithik96) [inline .github/workflows/maven-workflow-run.yml:106]: new line.

File: android/src/test/java/com/browserstack/AppiumTest.java (hunk 2 above)
Comment (karanshah-browserstack) [reply android/src/test/java/com/browserstack/AppiumTest.java:20]: Keeping consistent with parent selenide repo for now.

File: .github/workflows/maven-workflow-run.yml (hunk 3 above)
Comment (karanshah-browserstack) [reply .github/workflows/maven-workflow-run.yml:106]: Done

**Review by khrithik96 — APPROVED** [review]

**Review by darshilgada24 — APPROVED** [review]

**Pull request merged**